integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCustomSslStore.java

```diff
@@ -68,8 +68,8 @@
   private static String domainNamespace = null;
   private static int replicaCount = 2;
   private static final String domainUid = "mii-custom-ssl";
-  private static String pvName = domainUid + "-pv"; 
-  private static String pvcName = domainUid + "-pvc"; 
+  private static final String pvName = domainUid + "-pv"; // name of the persistent volume
```

Comment:
Why do not we use getUniquePvOrPvcName()  here ?

Reply:
made changes to create unique pv|pvc names